biscuit-auth/src/token/builder.rs

```diff
@@ -744,7 +754,10 @@ impl fmt::Display for Scope {
         match self {
             Scope::Authority => write!(f, "authority"),
             Scope::Previous => write!(f, "previous"),
-            Scope::PublicKey(pk) => write!(f, "ed25519/{}", hex::encode(pk.to_bytes())),
+            Scope::PublicKey(pk) => match pk {
+                PublicKey::Ed25519(key) => write!(f, "ed25519/{}", hex::encode(key.to_bytes())),
```

Comment:
see comment in `datalog/symbol.rs`

all this should be defined in a central location

Reply:
fixed in [b0490ae](https://github.com/biscuit-auth/biscuit-rust/pull/108/commits/b0490ae97caa6e442c02b357f1097d7657c8c440)